oozie-to-airflow/mappers/base_mapper.py

```diff
@@ -91,3 +91,8 @@ def copy_extra_assets(self, input_directory_path: str, output_directory_path: st
         :return: None
         """
         return None
+
+    @staticmethod
+    def common_imports() -> Set[str]:
+        # TODO: for now, just in case we add all el functions as imports. We can optimize it later
```

Comment:
I think, you should add this import to workflow
Please look at:
https://github.com/GoogleCloudPlatform/cloud-composer/blob/09569063ed46b5d2ec4ff9d50127b1f316619989/oozie-to-airflow/converter/primitives.py#L50-L56

Reply:
👍  - did not know it exists :) Thanks!